jso/apis/src/main/java/org/teavm/jso/webaudio/AudioWorker.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright 2015 Alexey Andreev.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.jso.webaudio;
+
+import org.teavm.jso.JSMethod;
+import org.teavm.jso.JSObject;
+import org.teavm.jso.JSProperty;
+import org.teavm.jso.dom.events.EventListener;
+
+/**
+ *
+ */
+public interface AudioWorker extends JSObject {
+    @JSProperty
+    AudioWorkerParamDescriptor[] getParameters();
+
+    @JSProperty(value = "onmessage")
+    void setOnMessage(EventListener event);
+
+    @JSProperty(value = "onmessage")
+    EventListener getOnMessage();
+
+    @JSProperty(value = "onloaded")
+    void setOnLoaded(EventListener event);
+
+    @JSProperty(value = "onloaded")
+    EventListener getOnLoaded();
+
+    @JSMethod
+    void terminate();
+
+    @JSMethod
+    void postMessage(Object message, Object... transfer);
```

Comment:
You can't pass java Objects to JS methods, I guess you should use `JSObject` instead, see [JSO manual(https://github.com/konsoletyper/teavm/wiki/Interacting-with-JavaScript)]. Also, varargs are not yet supported, sorry.